Server/database/query/Bills/getUserBillType.js

```diff
@@ -0,0 +1,9 @@
+const connection = require('../../connection');
+
+module.exports = (userId) => {
+  const sql = {
+    text: 'SELECT type FROM bill where users_id = $1',
```

Comment:
this query returns all the types of bills, so if we have 3 bills with water type for this user it will be return water 3 times, so you need to make `distinct`.

Reply:
i add distinct to query ` SELECT DISTINCT  type FROM bill where users_id = $1`